packages/neos-ui-redux-store/src/CR/Nodes/selectors.js

```diff
@@ -73,6 +73,7 @@ export const makeChildrenOfSelector = allowedNodeTypes => createSelector(
     .map(
         contextPath => $get(contextPath, nodesByContextPath)
     )
+    .filter(node => node)
```

Comment:
I do not understand yet how this change fixes the behavior. Do you have a hint / explanation on why this helps? :-)

Reply:
Sure. Without this line we get an immutablejs array full of `undefined`s, and (for some reason) when it compares two such arrays, with identical number of underineds it tells that they are difference, so forces a re-render.

Reply:
ah; I see! THX!